openhcl/underhill_core/src/nvme_manager.rs

```diff
@@ -191,13 +204,24 @@ pub struct NvmeManagerClient {
 impl NvmeManagerClient {
     pub async fn get_namespace(
         &self,
+        name: String,
         pci_id: String,
         nsid: u32,
     ) -> anyhow::Result<nvme_driver::Namespace> {
+        let params = GetNamespaceParams {
+            name: NvmeDeviceName(name),
```

Comment:
On second thought, the specific type names look weird. Since you have defined a `GetNamespaceParams` struct, this (and the `PciId`) can just be `String`.

In addition, let's change `name` to `debug_host_id` wherever you are making changes in `NvmeManagerClient`